web/crux-ui/src/models/notification.ts

```diff
@@ -9,19 +9,27 @@ export const NOTIFICATION_EVENT_VALUES = [
 ] as const
 export type NotificationEventType = typeof NOTIFICATION_EVENT_VALUES[number]
 
-export type CreateNotification = {
+export type Notification = {
+  id: string
   name: string
   url: string
   type: NotificationType
   active: boolean
+  createdBy: string
+}
+
+export type NotificationDetails = Notification & {
   events: NotificationEventType[]
 }
 
-export type UpdateNotification = CreateNotification & {
-  id: string
+export type CreateNotification = {
+  name: string
+  url: string
+  type: NotificationType
+  active: boolean
+  events: NotificationEventType[]
 }
 
-export type NotificationDetails = CreateNotification & {
+export type UpdateNotification = CreateNotification & {
   id: string
```

Comment:
The id should be in the url, therefor it does not need to be in the body.

Reply:
True, it's not even used :)